.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -16,6 +16,7 @@ trigger:
       - LICENSE
       - LICENSE-3rdparty.csv
       - NOTICE
+      - .gitlab-ci.yml
```

Comment:
I think it shouldn't be here (in the `master` exclude list), it should be in the PR exclude list. As for the fact that there are 2 very different lists... yeah...

Reply:
Yes I've done that way to quickly indeed. Thanks